### ISSUE TITLE
whitespace rule reports false positives inside import() types

### ISSUE THREAD

**Issue by karol-majewski** [palantir/tslint#4076]
### Bug Report

- __TSLint version__: 5.11.0
- __TypeScript version__: 2.9.2
- __Running TSLint via__: (pick one) CLI

#### TypeScript code being linted

```ts
interface Pusher {
  push: typeof import('react-router-redux').push;
}
```

with `tslint.json` configuration:

```json
{
  "rules": {
    "whitespace": [
      true,
      "check-preblock"
    ]
  }
}
```

#### Actual behavior

Error: `missing whitespace (whitespace)` right after the `import` keyword.

#### Expected behavior

No error. `TypeQuery` is not a block.

**Comment by JoshuaKGoldberg** [comment]
Duplicate of #3987.
